packages/sources/galaxy-test/src/endpoint/price.ts

```diff
@@ -0,0 +1,50 @@
+import { DEFAULT_WS_API_ENDPOINT } from '../config'
+import { WebSocketTransport } from '@chainlink/external-adapter-framework/transports'
+import {
+  PriceEndpoint,
+  priceEndpointInputParameters,
+} from '@chainlink/external-adapter-framework/adapter'
+import { getAccessToken } from '../util'
+import { AdapterRequestParams, PriceEndpointTypes } from '../types'
+
+// Split by the characters "_" and "/" as pairs are in this format: "markPrice_BTC/USD"
+const SPLIT_PAIR_REGEX = /[_/]+/
+
+const getPair = (params: AdapterRequestParams) => {
+  const { base = '', quote = '' } = params
+  return base && quote && `markPrice_${base.toUpperCase()}/${quote.toUpperCase()}`
+}
+
+export const priceTransport = new WebSocketTransport<PriceEndpointTypes>({
+  url: () => DEFAULT_WS_API_ENDPOINT,
```

Comment:
We can instead add WS_API_ENDPOINT to the custom settings with a default to what's now in this constant